vips/vip-isolated-pools-comptroller-upgrade/vip-comptroller-upgrade-testnet.ts

```diff
@@ -0,0 +1,64 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const COMPTROLLER_BEACON = "0xdddd7725c073105fb2abfcbdec16708fc4c24b74";
+const NEW_COMPTROLLER_IMPLEMENTATION = "0xa764a2eAc5C59DFb23E43850cBA89117f1c9f5AB";
+const ACM = "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA";
+const POOL_STABLECOIN = "0x10b57706AD2345e590c2eA4DC02faef0d9f5b08B";
+const POOL_DEFI = "0x23a73971A6B9f6580c048B9CB188869B2A2aA2aD";
+const POOL_GAMEFI = "0x1F4f0989C51f12DAcacD4025018176711f3Bf289";
+const POOL_LIQUID_STAKED_BNB = "0x596B11acAACF03217287939f88d63b51d3771704";
+const POOL_TRON = "0x11537D023f489E4EF0C7157cc729C7B69CbE0c97";
+const FAST_TRACK_TIMELOCK = "0x3CFf21b7AF8390fE68799D58727d3b4C25a83cb6";
+const CRITICAL_TIMELOCK = "0x23B893a7C45a5Eb8c8C062b9F32d0D2e43eD286D";
+const NORMAL_TIMELOCK = "0xce10739590001705F7FF231611ba4A48B2820327";
+
+interface GrantAccess {
+  target: string;
+  signature: string;
+  params: Array<string>;
+}
+
+const grantAccessControl = (contract: string) => {
+  const accessProposals: Array<GrantAccess> = [];
+  [NORMAL_TIMELOCK, FAST_TRACK_TIMELOCK, CRITICAL_TIMELOCK].map(target => {
+    accessProposals.push({
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [contract, "_setForcedLiquidation(address,bool)", target],
```

Comment:
In IL is without the underscore. See [here](https://github.com/VenusProtocol/isolated-pools/pull/305/files#diff-8ef9294768226293f278525b40998f11db7ee32464279f47ca7ad35294bb6a3fR1017)
```suggestion
      params: [contract, "setForcedLiquidation(address,bool)", target],
```

Reply:
Done [b8df51c](https://github.com/VenusProtocol/vips/pull/74/commits/b8df51c4150f0d803df18b2ca95b352d4dd56d5f)